src/schema/feeds.ts

```diff
@@ -885,9 +885,22 @@ export const resolvers: IResolvers<any, Context> = traceResolvers({
     },
     authorFeed: feedResolver(
       (ctx, { author }: AuthorFeedArgs, builder, alias) =>
-        builder.andWhere(`${alias}.authorId = :author`, {
-          author,
-        }),
+        builder
+          .addSelect(
+            `count(*) filter (where ${alias}."scoutId" = '${author}')`,
+            'isScout',
+          )
+          .addSelect(
+            `count(*) filter (where ${alias}."authorId" = '${author}')`,
+            'isAuthor',
+          )
```

Comment:
Not sure if we must have the additional selects when we can check if the user is the author or scout if the relevant id property has value.

Reply:
It could be for a different user, and either scout or author.
Not sure what the alternative is here.